### ISSUE TITLE
Service Bus Queues: custom properties string values containing quotes are mutated in the received message

### ISSUE THREAD

**Issue by eiliven** [Azure/azure-sdk-for-python#628]
String custom properties containing quotes are mutated in the received message. 'This "should" work fine' as a value for a custom property in the input message becomes '\\"This \\"should\\" work fine\\"' in the received message.

Python code to reproduce
```
from azure.servicebus import ServiceBusService
from azure.servicebus import Message

service_namespace = 'YOUR_SERVICE_NAMESPACE'
shared_access_key_name = 'KEY_NAME'
shared_access_key_value = 'KEY_VALUE'
sbs = ServiceBusService(service_namespace=service_namespace,
                        shared_access_key_name=shared_access_key_name,
                        shared_access_key_value=shared_access_key_value)

def test_message(custom_properties):
    out_message = Message()
    out_message.custom_properties = custom_properties
    sbs.send_queue_message('receipt-classification-input', out_message)

    in_message = sbs.receive_queue_message('receipt-classification-input')

    if out_message.custom_properties == in_message.custom_properties:
        print('Custom properties identical when received')
    else:
        print('Expected: ' + str(out_message.custom_properties) + ' got ' + str(in_message.custom_properties))


test_message({'property': 'This should work fine'})
test_message({'property': 'This "should" work fine'})  # return value for property is '\\"This \\"should\\" work fine\\"'
```

**Comment by lmazuel** [comment]
Thank you for your precise feedback! I take a look asap.

Regards,

**Comment by lmazuel** [comment]
The package 'azure-servicebus' was released yesterday as 0.20.3 with the fix.
Thank you for your patience and sorry for the delay!
